copis/settings.py

```diff
@@ -0,0 +1,41 @@
+# This file is part of COPISClient.
+#
+# COPISClient is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# COPISClient is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with COPISClient.  If not, see <https://www.gnu.org/licenses/>.
+
+from dataclasses import dataclass
+
+from enums import DebugEnv
+
+@dataclass
+class Settings:
+    debug_env: DebugEnv
```

Comment:
Could you add a short docstring here?

Reply:
I did. Or do you mean add to the doc string to describe the properties?

Reply:
What you have is good!